push-spire-server-image.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
can we move the Dockerfile and this script under `spire/` directory? We might have multiple Dockerfiles in this repo and it might become confusing to manage

Reply:
SGTM, let me change that now

Reply:
Actually we need to reference code that is currently in `cmd` to build the image, so this needs to be at the top level so that `cmd` can be included in the Docker build context